starlark/unpack.go

```diff
@@ -209,14 +209,24 @@ func UnpackPositionalArgs(fnname string, args Tuple, kwargs []Tuple, min int, va
 		return fmt.Errorf("%s: got %d arguments, want %s%d", fnname, len(args), atmost, max)
 	}
 	for i, arg := range args {
-		if err := unpackOneArg(arg, vars[i]); err != nil {
+		if err := UnpackOneArg(arg, vars[i]); err != nil {
 			return fmt.Errorf("%s: for parameter %d: %s", fnname, i+1, err)
 		}
 	}
 	return nil
 }
 
-func unpackOneArg(v Value, ptr any) error {
+// UnpackOneArg unpacks a single argument, represented as a Value, into
+// the given target variable. The target must be a pointer to a supported
+// type, or implement the Unpacker interface (see UnpackArgs for more
+// details, including what types are supported).
+//
+// UnpackOneArg is used by both UnpackArgs and UnpackPositionalArgs to handle
```

Comment:
// This function defines the unpack operation for a single value.
// The implementations of most built-in functions use [UnpackArgs] and 
// [UnpackPositionalArgs] to unpack a sequence of positional and/or
// keyword arguments.